### ISSUE TITLE
Jetpack Monitor: Downtime Notification Copy

### ISSUE THREAD

**Issue by benhuberman** [Automattic/jetpack#9224]
We were recently alerted to some minor issues in the copy of the Jetpack Monitor downtime notification (screenshot courtesy of Matt):
![image uploaded from ios](https://user-images.githubusercontent.com/9620684/38290722-84eced48-3791-11e8-8489-e9b662f397fa.png)

To smooth things up a bit, here's a proposed edit:
> Good news! [SITE NAME] is now up and running again.
> The site's downtime lasted [NUMBER OF MINUTES] minutes. We're happy to report your site was back online as of [TIME] on [DATE].
> Learn more about Jetpack Monitor.

I'm not sure who exactly is best positioned to make the changes; thanks to @joanrho, who pointed me in the right direction on OpenGrok, it seems like @enejb may have been the last person to touch this particular notification, but @kwight has most recently worked on other notifications here, so... hopefully one of you might be able to edit the copy or know who else to ask? Thanks in advance.

Let me know if you have any questions / if the proposed copy creates any unforeseen issues.

**Comment by jeherve** [comment]
cc'ing @roccotripaldi who recently worked on the copy in those notifications as part of #7892

**Comment by kwight** [comment]
I can patch it up.

**Comment by benhuberman** [comment]
Thank you all!

**Comment by kwight** [comment]
Working in D11514-code. 

![screen shot 2018-04-04 at 11 10 07 am](https://user-images.githubusercontent.com/349751/38326075-b1fabcbc-37f9-11e8-87a0-3f997618f0b9.png)

@benhuberman any comments for the summary text to the right?

Note I also need to add a filter for `human_time_diff`, assuming we prefer "minutes" to "mins"?

**Comment by benhuberman** [comment]
Yup, "mins" to "minutes" is the main thing left to fix here, I'd say (both in the main body and the summary on the right side). 

The other small thing in the right-side summary is that both sentences there are missing terminal punctuation. I'd go with either periods for both sentences, or an exclamation point / period combo.

**Comment by kwight** [comment]
@benhuberman 💥 

![screen shot 2018-04-04 at 1 02 10 pm](https://user-images.githubusercontent.com/349751/38331739-e324c06c-3808-11e8-8afc-1d9be1620139.png)

**Comment by kwight** [comment]
When I've got an approval for d11514-code, I'll push it out.
